deployment/volumes/volume-snapshots/README.md

```diff
@@ -9,13 +9,27 @@ Kubernetes announced this feature in a [blog post](https://kubernetes.io/blog/20
 ## Create a Volume Snapshot
 
 There is an example in this folder how you can e.g. create a volume snapshot for
-the persistent volume claim `neo4j-data-claim`:
+the persistent volume claim `neo4j-data-claim` or any other claim.
+
+Before you create the snapshot, it is good practice to look at what snapshots are already done:
+
+```sh
+# list existing snapshots
+kubectl get volumesnapshots -n human-connection
+```
+
+To specify your snapshot, edit the file `snapshot.yaml` or simply edit and use `neo4j-data.yaml` in this folder.
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-08T13:17:02Z" title="Wednesday, April 8th 2020, 3:17:02 pm +02:00">Apr 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -9,13 +9,27 @@ Kubernetes announced this feature in a [blog post](https://kubernetes.io/blog/20
 ## Create a Volume Snapshot
 
 There is an example in this folder how you can e.g. create a volume snapshot for
-the persistent volume claim `neo4j-data-claim`:
+the persistent volume claim `neo4j-data-claim` or any other claim.
+
+Before you create the snapshot, it is good practice to look at what snapshots are already done:
+
+```sh
+# list existing snapshots
+kubectl get volumesnapshots -n human-connection
+```
+
+To specify your snapshot, edit the file `snapshot.yaml` or simply edit and use `neo4j-data.yaml` in this folder.
```


you don't want to ever edit and apply the `neo4j-data.yaml`! this is for creating new `volumes` not for anything to do with `volumesnapshots`

you would probably get an error that the `neo4j-data-claim` already exists, but still there is no need to ever apply this file, unless you are setting up a new `persistent volume`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-04-15T09:15:09Z" title="Wednesday, April 15th 2020, 11:15:09 am +02:00">Apr 15, 2020</time>_

---


I have to speak about this to you in person, because I get not safely the point. @mattwr18

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-16T06:25:56Z" title="Thursday, April 16th 2020, 8:25:56 am +02:00">Apr 16, 2020</time>_

---


we can do that, the point is that you should not edit the `neo4-data.yaml` since it is for setting up `PersistentVolumeClaims`/`PersistentVolumes`, not for creating a `VolumeSnapshot`.
Also, it's potentially dangerous.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-04-16T07:19:54Z" title="Thursday, April 16th 2020, 9:19:54 am +02:00">Apr 16, 2020</time>_

---


Okay, I get this.
Thanks! @mattwr18